src/components/Category.jsx

```diff
@@ -0,0 +1,95 @@
+import adidasShoes from "../assets/adidas-shoes.jpg";
+import nikeShoes from "../assets/nike-shoes.jpg";
+import newBalanceShoes from "../assets/new-b-shoes.jpg";
+import hAndMShirt from "../assets/h&m-shirt.webp";
+import levisShirt from "../assets/levis-shirt.webp";
+import wBrand from "../assets/w-kurta.webp";
+import womenKurta from "../assets/women-kurta.webp";
+import newBalanceTShirt from "../assets/new-balance-t-shirt.jpg";
+import uspaShirt from"../assets/uspa-shirt.webp";
+import { ProductsListPage } from "../pages/ProductsListPage";
+import { Link } from "react-router-dom";
+
+export const Category = () =>{
+    return(
+        <div>
+            <main className="main-login main-home">
+                <div className="category-header">
+                <h2 className="brand-heading">
+                    Women's Clothing
+                </h2 >
+                </div>
+                <div className="sale-img-container">
+                    <div>
+                        <Link to="/products" element={<ProductsListPage/>}>
```

Comment:
You don't need to specify the element in the `Link` component, it is specified in the route component.

Reply:
These changes have been taken care of. Please review the changes and approve them. Thank you for your feedback🌻